typescript/sdk/src/consts/chainMetadata.ts

```diff
@@ -17,6 +17,12 @@ export enum ExplorerFamily {
   Other = 'other',
 }
 
+export enum ProtocolType {
+  Ethereum = 'ethereum',
```

Comment:
I also called this 'Ethereum' because that's what the agent did but 'EVM' feels more correct. Now may be a reasonable time to rename it throughout if others agree. @tkporter @nambrot

Reply:
imo the pain of a breaking change like this outweighs the benefit of the rename